### PR TITLE
man: update InitialCongestionWindow= and InitialAdvertisedReceiveWindow=

### LINKED ISSUE #16643: InitialCongestionWindow and InitialAdvertisedReceiveWindow are multipliers for MSS
**systemd version the issue has been seen with**
> v246

**remainder of template irrelevant, as this is a documentation bug**

The man page for systemd.network describes the value for `InitialCongestionWindow` and `
`InitialAdvertisedReceiveWindow` as

>  Takes a size in bytes between 1 and 4294967295 (2^32 - 1). The usual suffixes K, M, G are supported and are understood to the base of 1024.

This is not accurate.

A correct description of the value, from the `iproute2` documentation:

> Actual window size is this value multiplied by the MSS (``Maximal Segment Size'') for same connection.

Since MSS may be affected by various conditions on a per connection basis, it is not possible to compute a value to use for the route from a number of bytes.

As such:

*  Support for K/M/G suffixes should be removed.

* Documentation should be updated with wording similar to that used by `iproute2`

### PR DESCRIPTION
**Pull request by yuwata**
Fixes #16643.

cc @ryancdotorg.

### CODE REVIEW

**Comment by keszybz** [comment]
I don't think it makes much sense to keep backwards compat for this. If someone wrote a large number (either directly or using a multiplier suffix), this is almost certainly because they were mislead by our faulty documentation. In that scenario they would be better off if we ignored the option, rather than as "honouring" the option in a way that breaks their networking. 

Based on @ryancdotorg linked, 1k would be considered far outside of the realm of useful values. So anything with "k" suffix is already broken, and with "M" even more so. I think we should simply reject any numbers equal to or higher than 1024.

**Comment by yuwata** [comment]
@keszybz Thank you for the comment, and sorry for late response. Now the value is limited from the above with 1024. And k, M, G suffixes support is dropped. PTAL.

File: man/systemd.network.xml [hunk 1]
```diff
@@ -1389,21 +1389,22 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
...
           <listitem>
-            <para>The TCP initial advertised receive window is the amount of receive data (in bytes) that can initially be buffered at one time
-            on a connection. The sending host can send only that amount of data before waiting for an acknowledgment and window update
-            from the receiving host. Takes a size in bytes between 1 and 4294967295 (2^32 - 1). The usual suffixes K, M, G are supported
-            and are understood to the base of 1024. When unset, the kernel's default will be used.
-            </para>
+            <para>The TCP initial advertised receive window is the amount of receive data (in bytes)
+            that can initially be buffered at one time on a connection. The sending host can send only
+            that amount of data before waiting for an acknowledgment and window update from the
+            receiving host. Takes a size in bytes between 1 and 1023. Note that 100 is considered an
```
Comment (keszybz) [inline man/systemd.network.xml]: "Takes a number between 1 and 1023."

**Review by keszybz — CHANGES_REQUESTED** [review]

**Comment by yuwata** [comment]
@keszybz Thanks. Updated. Setting the green label.

**Comment by keszybz** [comment]
bionic-i386 timed out in boot-smoke. Let's assume this is unrelated unless something else fails.

**Pull request merged**